README.md

```diff
@@ -244,71 +271,75 @@ AwsSsmEnv.load!(
 
 #### other
 
-`AwsSsmEnv::NamingStrategy`を実装したクラスのインスタンス、もしくは`parse_name`メソッドを持つ  
-インスタンスを指定した場合はそのインスタンスをそのまま利用する。
+If you specify an instance of a class that implements `AwsSsmEnv::NamingStrategy` in` fetch`,
+or an instance with a `parse_name` method, use that instance as is.
+
 
 ### path: [String]
 
-`fetch`に何も指定していない場合、もしくは`:path`を指定した場合は必須となる。  
-パラメータを取得するパス階層を指定する。  
-下の例では`/myapp/web/production`直下のパラメータが取得される。
+It is required if nothing is specified for `fetch` or if `:path` is specified.  
+Specify the path hierarchy for acquiring parameters.  
+In the example below, the parameter immediately under `/myapp/web/production` is acquired.
 
 ```ruby
 AwsSsmEnv.load(path: '/myapp/web/production')
 ```
 
 #### recursive: [Boolean]
 
-`fetch`に何も指定していない場合、もしくは`:path`を指定した場合に利用する。  
-指定したパス階層以下のパラメータをすべて取得する。  
-下の例では`/myapp/web/production`以下すべてのパラメータが取得される。
+It is required if nothing is specified for `fetch` or if `:path` is specified.  
+Acquires all parameters below the specified path hierarchy.  
+In the following example, all parameters below `/myapp/web/production` are acquired.
 
 ```ruby
 AwsSsmEnv.load(path: '/myapp/web/production', recursive: true)
 ```
 
 ### begins_with: [String, Array<String>]
 
-`fetch`に`:begins_with`を指定した場合は必須となる。  
-取得するパラメータ名のプレフィクスを指定する。配列で複数指定することも可能(OR条件となる)。  
-下の例では`myapp.web.production`で始まる名前のパラメータが取得される。
+It is required if `:begins_with` is specified in `fetch`.  
+Specify the prefix of the parameter name to be acquired.
+It is also possible to specify more than one in an array (OR condition).    
+In the example below, parameters with names starting with `myapp.web.production` are acquired.
 
 ```ruby
 AwsSsmEnv.load(path: 'myapp.web.production')
 ```
 
 ### removed_prefix: [String]
 
-`naming`に`:snakecase`を指定した場合に利用される。  
-環境変数名から除外するパラメータ名のプレフィクスを指定する。  
-`:removed_prefix`が指定されておらず、`:begins_with`もしくは`:path`が指定されていた場合はそれを利用する。
+It is used when `:snakecase` is specified in `naming`.  
+Specify the prefix of the parameter name to exclude from the environment variable name.  
+If `:removed_ prefix` is not specified, and `:begins_with` or `:path` was specified, use it.
 
 ### delimiter: [String, Regexp]
 
-`naming`に`:snakecase`を指定した場合に利用される。  
-アンダースコアに変換する文字列もしくは正規表現を指定する。  
-デフォルトはスラッシュ(`/`)。
+It is used when `:snakecase` is specified in `naming`.  
+Specify a character string or regular expression to be converted to an underscore.  
```

Comment:
The same two points with above are also applicable here.

For example:

> Used when `:snakecase` is specified in `naming`. 
By this option, you can specify a string or a regular expression to be converted to an underscore.